src/base/bittorrent/torrenthandleimpl.cpp

```diff
@@ -420,6 +420,10 @@ void TorrentHandleImpl::replaceTrackers(const QVector<TrackerEntry> &trackers)
             newTrackers << tracker;
     }
 
+    // Do not allow tracker merging on private tracker.
```

Comment:
I think this action should be done at UI level, not in core.

Reply:
The problem with that is we have another edit tracker dialogue in transfer list where you can add multiple trackers, tiers and or remove all trackers. We’ll have to change that thing as well then if we don’t want to allow users to add new trackers/tiers or delete the last tracker.